### DISCUSSION TITLE
How to migrate from UWP to WinUI for apps that use WebView?

### DISCUSSION THREAD

**Discussion by jeanplevesque** [unoplatform/uno#10955] [Q&A] [answered]
With Uno.WinUI, `WebView` is still exposed for iOS and Android, but not for Windows.

![image](https://user-images.githubusercontent.com/39710855/211841740-5757d46a-12fe-48d6-8582-b08ae6abb910.png)

What would be the unified migration path?

**Comment by jeromelaban** [comment, marked as answer]
The migration path will be to use WebView2, which is in the works in #10794.
